### PR TITLE
Add default tab on downloads page based on user OS

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Checks the users operating system and sets this as the default tab when navigating to the downloads page.

## Preview link
https://deploy-preview-167--melodious-puffpuff-8e1109.netlify.app/downloads

### CODE REVIEW

File: src/components/UI/downloads/DownloadsTable.tsx [hunk 1]
```diff
@@ -37,6 +37,15 @@ export const DownloadsTable: FC<Props> = ({
 
   const LAST_2_LINUX_RELEASES = amountOfReleasesToShow + 12;
 
+  const getDefaultIndex = useMemo<number>(() => {
+    const OS: string = typeof window !== 'undefined' ? window.navigator.platform : '';
```
Comment (nhsz) [inline src/components/UI/downloads/DownloadsTable.tsx:41]: I think this check doesn't work on mobile browsers

**Review by nhsz — CHANGES_REQUESTED** [review]
the OS detection doesn't work on mobile, we should [check userAgent instead](https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator)

**Comment by wackerow** [comment]
It's working for me on iPhone, I take it Android is not? Can look into it shortly

File: src/components/UI/downloads/DownloadsTable.tsx (hunk 1 above)
Comment (wackerow) [reply src/components/UI/downloads/DownloadsTable.tsx:41]: Gotcha... okay, it seems to be working on iOS, but I'll look into it

**Comment by wackerow** [comment]
So I don't think we can only use the user agent, since we're looking for the operating system not the browser. I can confirm this works on Linux, Windows, macOS and iOS... if it isn't working on Android, I can add a fallback to also check for `userAgent.includes("SamsungBrowser")` and default to Android in that case.

**Review by nhsz — APPROVED** [review]

**Comment by wackerow** [comment]
@nhsz I didn't have luck when testing with BrowserStack using just the SamsungBrowser suggested in that mdn link, but checking for "Android" within user agent seems to work. 

Just double checked that making the OS check case-insensitive didn't fix this, which it didn't. Removed the OS check for "Android" and replaced with a user agent check for "Android" instead which seems to do the trick:

image

Should be good to merge now.

**Pull request merged**